Tracking/TrackLength/README.md

```diff
@@ -0,0 +1,126 @@
+## Description
+
+TrackLength is the Marlin processor that calculates track length and square root of the harmonic mean of the squared momentum of charged particles.
+
+The detailed description of the source code is available in the [doxygen documentation](https://www.desy.de/~dudarboh/track_length_doc/html/index.html). <br>
```

Comment:
Once the doxygen generation is fixed, this should probably point to a more central location.